.github/workflows/vuln-check.yaml

```diff
@@ -14,18 +14,39 @@ on:
         type: boolean
         default: false
     secrets:
-      CR_PAT:
-        required: true
       SLACK_SECURITY_WEBHOOK_URL:
         required: false
 
 jobs:
-  call-vuln-check:
-    uses: scalar-labs/actions/.github/workflows/vuln-check-reusable.yaml@main
-    with:
-      target-ref: ${{ inputs.target-ref }}
-      find-latest-release: ${{ inputs.find-latest-release }}
-      images: '[["ScalarDL Ledger", "scalardl-ledger"], ["ScalarDL Client", "scalardl-client"]]'
-    secrets:
-      CR_PAT: ${{ secrets.CR_PAT }}
-      SLACK_SECURITY_WEBHOOK_URL: ${{ secrets.SLACK_SECURITY_WEBHOOK_URL }}
+  vuln-check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Build docker images
+        id: build-docker-images
+        uses: scalar-labs/actions/.github/actions/build-docker-images@vuln-check-composite-actions
```

Comment:
@ymorimo I don't know the composite actions very much. The latter part of `@` is the branch name to specify which version of the actions is used? Can we omit it (or specify the master/main) to use the latest? I'm concerned about the maintainability when the actions are updated.

Reply:
Is the above thing mentioned in the additional notes?

Reply:
@feeblefakie @ymorimo Oh, sorry, I missed it. Got it, thank you!